.github/workflows/install-shared-dependencies/action.yml

```
@@ -21,6 +21,12 @@ inputs:
             - aarch64-apple-darwin
             - aarch64-unknown-linux-musl
             - x86_64-unknown-linux-musl
+    install-redis:
+        description: "Indicates if redis should be installed from the package manager"
+        required: false
+        type: boolean
+        default: "true"
```

This probably should contain the version to be installed or none/null if is not needed
```suggestion
        type: string
        default: null
```